README.md

```diff
@@ -32,6 +32,8 @@ General guidelines for Hellbender developers
 
 * Untested code is considered **non-existent** and thus is subject to removal at any time (exception is main methods, or super corner conditions, or `toString()` code).
 
+* Each tool should have at least one good end-to-end integration test with a check for expected output, plus high-quality unit tests for all non-trivial utility methods/classes used by the tool. Although we have no specific coverage target, coverage should be extensive enough that if tests pass, the tool is guaranteed to be in a usable state.
```

Comment:
I agree that each tool should have at a minimum one integration test.

I'm going to merge this pull req. The discussion on what tools need what tests was moved to a spreadsheet for convenience. @nh13 and company will populate the issue list from that spreadsheet